### DISCUSSION TITLE
Consider creating your own .github repo

### DISCUSSION THREAD

**Discussion by jcbhmr** [spenserblack/spenserblack#44] [General]
it's a pretty great feature! you get:
- the ability to add a `<!-- ❤️ Thanks for contributing! -->` thing to a global PULL_REQUEST_TEMPLATE.md
- the ability to add custom workflow templates that appear in the actions web wizard https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization
- the ability to set a global CONTRIBUTING.md for all repos (to give generic friendly contrib instructions)

my personal favorite highlight is the `<!-- ❤️ Thanks for contributing! -->` thing. in an odd way, it's kinda nice to have that even when you open a pr to one of your own repos 🤗

**Comment by spenserblack** [comment]
I didn't realize this worked for personal repos. I thought it was only for orgs. Thanks for letting me know!
